package/utils/busybox/files/sysntpd

```diff
@@ -4,7 +4,7 @@
 START=98
 
 USE_PROCD=1
-PROG=/usr/sbin/ntpd
+PROG=ntpd
```

Comment:
jail does not allow this.
procd fails like this:
````
Sun Mar  5 23:08:39 2023 user.err : jail: ntpd is not an absolute path
````

Reply:
How about this?
PROG=$(which ntpd)

If not, BUSYBOX_CONFIG_INSTALL_NO_USR should be permanently removed.